nuopc.runconfig

```diff
@@ -284,17 +284,17 @@ CLOCK_attributes::
      glc_avg_period = yearly
      glc_cpl_dt = 86400
      history_ymd = -999
-     ice_cpl_dt = 3600
+     ice_cpl_dt = 1350
      lnd_cpl_dt = 3600
-     ocn_cpl_dt = 3600
+     ocn_cpl_dt = 1350
```

Comment:
A quick note on these. As was mentioned in [https://github.com/COSIMA/access-om3/wiki/NUOPC-driver#time-steps](https://github.com/COSIMA/access-om3/wiki/NUOPC-driver#time-steps), for our current NUOPC config, `{model_name}_cpl_dt` are unused. 

> So I think in our case, {model_name}_cpl_dt are unused and the driver time-step equals the coupling time-step set in nuopc.runseq. Certainly, changing these values seems to have no effect.

Reply:
Yeah. I wonder if we should change these to something obviously ridiculous (in another PR) to make it clear to our future selves that they aren't used. E.g. I wrote that documentation in the wiki, but I forgot that they aren't used when I was reviewing this PR... 😅

Reply:
I've open this issue so we can come back to this: https://github.com/COSIMA/access-om3/issues/132